src/swe/swe_ceed_impl.h

```diff
@@ -236,14 +183,18 @@ CEED_QFUNCTION(SWEBoundaryFlux_Outflow_Roe)(void *ctx, CeedInt Q, const CeedScal
 
   for (CeedInt i = 0; i < Q; i++) {
     CeedScalar sn = geom[0][i], cn = geom[1][i];
-    SWEState   qL    = {q_L[0][i], q_L[1][i], q_L[2][i]};
+    SWEState   qL    = {.h = q_L[0][i], .hu = q_L[1][i], .hv = q_L[2][i]};
```

Comment:
Note that designated initializers were introduced in C99 and C++20, so this may reduce portability.

Reply:
Indeed. A lot of other stuff we're using is also C99 (like C++ style comments and variable declarations not at the top of functions), so I'm not worried about this too much.

Reply:
If you're using a C++ compiler, as occurs when building CUDA or HIP kernels, it needs to be a C++ dialect that supports this syntax. That didn't come officially until C++20. If it's working on relevant machines for RDycore, that's fine. It wouldn't be allowed in PETSc because too many users are on old toolchains.

Reply:
Ah, I see. I can back out these changes, then.